test/test_infer_states_optimized.py

```diff
@@ -65,7 +65,7 @@ def setUpClass(cls):
         parameter_sets = [
             (5, 5, 5, 5, 'uniform', 'low'),
             (10, 10, 10, 10, 'uniform', 'medium'),
-            (25, 25, 25, 25, 'uniform', 'high'),
+            # (25, 25, 25, 25, 'uniform', 'high'),
```

Comment:
I believe it's this level of model complexity that was causing unit tests to fail on my local machine and on CI - needing too much memory

I think long term we should have separate benchmark tests that live outside the unit tests and run on a different cadence